src/contentmanagerdelegate.cpp

```diff
@@ -253,7 +253,8 @@ bool ContentManagerDelegate::editorEvent(QEvent *event, QAbstractItemModel *mode
         const auto lastColumnClicked = ((index.column() == 5) && (clickX > x && clickX < x + w)
                                                         && (clickY > y && clickY < y + h));
 
-        if (lastColumnClicked)
+        /* The description row does not constitute as part of the open button. */
+        if (lastColumnClicked && !index.parent().isValid())
             handleLastColumnClicked(index, e, option);
```

Comment:
I [fixed this same problem slightly differently](https://github.com/kiwix/kiwix-desktop/pull/1118/commits/aae0fd5de2464f5d607141f2093f2de1e20cc819) in my so far WIP PR #1118. Do you think that an early return may have undesirable effects?

Reply:
An early return would indeed fix this. I will remove my line then.

Reply:
@veloman-yunkan I believe fixing this your commit makes more sense. Your can do that check anytime after the middle button click check.